HarvestMobile/App.tsx

```diff
@@ -1,19 +1,31 @@
 import { StatusBar } from 'expo-status-bar';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import * as WebBrowser from "expo-web-browser";
 
 import useCachedResources from './hooks/useCachedResources';
 import useColorScheme from './hooks/useColorScheme';
 import Navigation from './navigation';
+import i18n from './services/i18n';
+import { useState } from '@hookstate/core';
+import { useLocale } from './hooks/useLocale';
 
 WebBrowser.maybeCompleteAuthSession();
 
 export default function App() {
   const isLoadingComplete = useCachedResources();
   const colorScheme = useColorScheme();
+  const isI18nInitialized = useState(false);
 
-  if (!isLoadingComplete) {
+  useEffect(() => {
+    i18n.init()
+      .then(() => {
+        isI18nInitialized.set(true);
```

Comment:
is this a thing now instead of doing `const [isI18nInitialized, setIsI18nInitialized] = useState(false)`?

Reply:
It is for hook-state. Unfortunate that they didn't choose a different name, such as `useHookState`.

Reply:
oh that sucks